lib/index.js

```diff
@@ -38,21 +40,36 @@ function renderSongs(data) {
   for (var track in tracks) {
    let song = tracks[track]["track"]
    var veriGenre = genre(song)
-   
-   var par= document.createElement("P")
-   var t = document.createTextNode(`Title: ${song.track_name}`)
-   par.appendChild(t)
-   document.getElementById("song").appendChild(par);
-   var par= document.createElement("P")
-   var t = document.createTextNode(`Genre: ${veriGenre}`)
-   par.appendChild(t)
-   document.getElementById("song").appendChild(par);
-
-   var button= document.createElement("BUTTON")
-   var t = document.createTextNode("Favorite")
-   button.appendChild(t)
-   document.getElementById("song").appendChild(button);
+   const element = `
+     <div class="song">
+       <p>Title: ${song.track_name}</p>
+       <p>Genre: ${veriGenre}</p>
+       <button type="button" name="button">
+        <i class="far fa-star"></i>
+       </button>
+     </div>
+   `;
+
+   const div = document.createElement('div')
+   div.innerHTML = element;
+   document.getElementById("songs").appendChild(div);
   }
 }
+function renderFavorites(data){
+  console.log(data)
+}
+
+function getFavorites() {
+  fetch(`https://evening-cliffs-86902.herokuapp.com/api/v1/favorites`)
+    .then(response => response.json())
+    .then(data => renderFavorites(data))
+    .catch((error) => console.error({ error }))
+}
+
+function showFavorites() {
+  getFavorites()
+}
```

Comment:
Considering this function only exists to call another one - I question it's purpose